extension/sumologicextension/extension.go

```diff
@@ -715,13 +715,14 @@ var sumoAppProcesses = map[string]string{
 func filteredProcessList() ([]string, error) {
 	var pl []string
 
-	p, err := ps.Processes()
+	p, err := process.Processes()
 	if err != nil {
 		return pl, err
 	}
 
 	for _, v := range p {
-		e := strings.ToLower(v.Executable())
+		e, _ := v.Name() // Ignore error
+		e = strings.ToLower(e)
```

Comment:
Why do we ignore the error? What are possible errors?

Reply:
Looks like, at least on some platforms, calling Name() can result in a syscall, so I guess the error is platform specific. IMO we should propagate these errors to the caller.